core/src/main/java/com/novoda/noplayer/model/AudioTracks.java

```diff
@@ -0,0 +1,78 @@
+package com.novoda.noplayer.model;
+
+import com.novoda.noplayer.internal.exoplayer.mediasource.AudioTrackType;
+import com.novoda.utils.Optional;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class AudioTracks implements Iterable<PlayerAudioTrack> {
+
+    private final List<PlayerAudioTrack> audioTracks;
+
+    public static AudioTracks from(List<PlayerAudioTrack> audioTracks) {
+        return new AudioTracks(Collections.unmodifiableList(audioTracks));
+    }
+
+    private AudioTracks(List<PlayerAudioTrack> audioTracks) {
+        this.audioTracks = audioTracks;
+    }
+
+    public boolean containsPlayerAudioTrackWith(AudioTrackType trackType) {
+        for (PlayerAudioTrack audioTrack : audioTracks) {
+            if (audioTrack.audioTrackType() == trackType) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public Optional<PlayerAudioTrack> firstPlayerAudioTrackWith(AudioTrackType audioTrackType) {
+        for (PlayerAudioTrack audioTrack : audioTracks) {
+            if (audioTrack.audioTrackType() == audioTrackType) {
+                return Optional.of(audioTrack);
+            }
+        }
+        return Optional.absent();
+    }
+
+    public PlayerAudioTrack getPlayerAudioTrackAt(int position) {
```

Comment:
`get(int position)` is idiomatic enough for a collection type? It'll be `audioTracks.get(i)` vs `audioTracks.getPlayerAudioTrackAt(i)`